site/docs/v1/tech/identity-providers/openid-connect/cognito.adoc

```diff
@@ -0,0 +1,135 @@
+---
+layout: doc
+title: Amazon Cognito OpenID Connect
+description: Login using Cognito as an OpenID Connect Identity Provider
+navcategory: login-methods
+---
+
+:sectnumlevels: 0
+
+== Configure OpenID Connect with Cognito
+
+[NOTE]
+====
+Cognito users must have a public email address configured to link on email (See link:/docs/v1/tech/identity-providers#linking-strategies[Linking Strategies for more on this]). An approach that will work for all users is to link on username or create an anonymous link. Using these strategies ensures that the configured Cognito OIDC connection works for every user, no matter their Cognito privacy settings.
+====
+
+Once you have completed this configuration, you may enable an OpenID Connect "Login with Cognito" button for one or more FusionAuth Applications. See link:https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html[Cognito - Configuring a user pool app client] for an additional reference.
+
+image::identity-providers/cognito/cognito-login-fusionauth-button.png[Login with Cognito, width=1200, role=bottom-cropped]
+
+=== Register a Cognito User Pool
+
+If you're interested in connecting to Cognito, it is likely that you already have a user pool set up that you'd want to connect to. We're adding the steps to create a new user pool in this guide in the interest of completeness, or in case you would like to set up a test user pool.
```

Comment:
Is there a document on the AWS side you can link to on setting up a user pool? That might be clearer and kept up to date.